### PR TITLE
Don't update XAML version

### PR DESCRIPTION
**Pull request by genlu**
Issuer is fixed in [this PR](https://devdiv.visualstudio.com/DevDiv/_git/a290117c-5a8a-40f7-bc2c-f14dbe3acf6d/pullrequest/558810)

### CODE REVIEW

**Review by arkalyanms — APPROVED** [review]

File: src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config [hunk 1]
```diff
@@ -94,7 +94,7 @@
         <value>False</value>
       </setting>
       <setting name="UpdateXamlRoslynVersion" serializeAs="String">
-        <value>True</value>
+        <value>False</value>
```
Comment (jjonescz) [inline src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config:97]: Should we update also `src/RoslynInsertionTool/RoslynInsertionTool.Commandline/Settings.settings`?

File: src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config
Comment (jjonescz) [inline src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config:1]: > Issuer is fixed in [this PR](https://devdiv.visualstudio.com/DevDiv/_git/a290117c-5a8a-40f7-bc2c-f14dbe3acf6d/pullrequest/558810)

Are we sure it's fixed? It doesn't look like they tried inserting with the version not matching. But I guess we will see and can always change the default back.

**Review by jjonescz — APPROVED** [review]

File: src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config
Comment (genlu) [reply src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config:1]: Yea, I tried to test their fix in this PR, which still failed with build error. I ask them to take a look
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/560567

Comment (genlu) [reply src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config:1]: Tested again, now it seems to be fixed
https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/563574

**Pull request merged**